gadopt/free_surface_equation.py

```diff
@@ -1,48 +1,55 @@
-from .equations import BaseTerm, BaseEquation
-from firedrake import dot, FacetNormal
-r"""
-This module contains the free surface terms and equations
+r"""This module contains the free surface terms.
 
-NOTE: for all terms, the residual() method returns the residual as it would be on the RHS of the equation, i.e.:
+All terms implement the UFL residual as it would be on the RHS of the equation:
 
-  dq/dt = \sum term.residual()
+  dq/dt = \sum term
 
-This sign-convention is for compatibility with Thetis' timeintegrators. In general, however we like to think about
-the terms as they are on the LHS. Therefore in the residual methods below we assemble in F as it would be on the LHS:
+This sign-convention is for compatibility with Thetis's time integrators. In general,
+however, we like to think about the terms as they are on the LHS. Therefore, in the
+function below, we assemble in F as it would be on the LHS:
 
   dq/dt + F(q) = 0
 
-and at the very end "return -F".
+and at the very end return "-F".
 """
 
+import firedrake as fd
```

Comment:
again slightly different to what was before and in momentum/scalar...

Reply:
Given the comment in `momentum_equation.py`, this module can serve as an example documenting what the source code looks like when `import firedrake as fd` is used.